ripple/src/main/webapp/businessdetails.html

```diff
@@ -288,8 +288,113 @@ <h3 class="col-12 modal-title bd-center-align">
         </a>
       </div>
       <!-- ./Order links -->
+      <!-- User reviews -->
+      <div class="bd-padding-top"></div>
+      <hr class="line-break">
+      <div class="padding-top"></div>
+      <div id="user-reviews" class="container">
+        <!-- Single user review -->
+        <div id="single-user-review" class="margin-top-lg">
+          <div class="inline-block-child width-100">
+            <!-- User information and rating. Delete button -->
+            <div class="width-100 opp-elements">
+              <p class="inline-block-child">
+                <span><img class="inline-block-child-no-padding review-avatar" src="/images/avatar_stock_man.jpg"></span>
+                <span id="review-user-name" class="boldest-text inline-block-child">Sarah Wu</span>  
+                <span id="review-time-stamp margin-left">August 3rd, 2020</span>
+              </p>
+              <div class="inline-block-child">
+                <div class="inline-block-child star-color">
+                  <span id="review-star-1" class="fa fa-star fa-lg"></span>
+                  <span id="review-star-2" class="fa fa-star fa-lg"></span>
+                  <span id="review-star-3" class="fa fa-star fa-lg"></span>
+                  <span id="review-star-4" class="fa fa-star fa-lg"></span>
+                  <span id="review-star-5" class="fa fa-star fa-lg"></span>
+                </div>
+                <div class="inline-block-child">
+                  <div class="inline-block-child"><div class="vl"></div></div>
+                  <p class="inline-block-child" id="review-price">
+                    <span id="review-price-1">$</span><span id="review-price-2">$</span><span id="review-price-3">$</span>
+                  </p>
+                  <i class="inline-block-child-no-padding cursor-pointer margin-left fas fa-trash-alt fa-lg"></i>
+                </div>
+              </div>
+            </div>
+            <!-- ./User information and rating. Delete button -->
+          </div>
+          <!-- Review text -->
+          <div>
+            <p class="review-text margin-left-lg">
+              I love this restaurant so much! The following setences repeat.
```

Comment:
Just wondering, is there a character limit imposed in the review form? If not we should probably add one.

Reply:
Good point. Will add in a future PR

Reply:
Added a char limit of 5000. That's about 500-1000 words, which is similar to how other companies limit their reviews input.